pre-docs/inventory_version2.md

```diff
@@ -90,8 +95,14 @@ groups:
     target-lookups:
       - plugin: puppetdb
         query: "inventory[certname] { facts.osfamily = 'RedHat' }"
+        config:
```

Comment:
This example won't work for two reasons:

1. the config is not scoped to a transport so I think it will be ignored.
2. host won't be used because uri will be magically set.

I think we should leave the simple example where config is not set and then add a new more complex one that uses an ip address as the host.  We should also explain the magic and what values are available, (facts and certname)

```
name: certname
config:
   ssh:
     host: facts.networking.interfaces.en0.ipaddress
     user: facts.identity.user
```

Reply:
I have tested the example i linked in https://github.com/puppetlabs/bolt/pull/993#discussion_r286259187 against PE nodes. I use `facts.networking.hostname` for the `host` and  `facts.ipaddress` for the target `name`. It seems like you are suggesting the opposite. I am wondering if there is something i have set in OpenSSH config that is making this magically work for me.

Reply:
that will work as long as the hostname is resolvable. The goal with my example is to connect to a node with an ip address without relying on dns.